UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,659 @@
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "other",
+                "display": "Other"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "3",
+      "type": "group",
+      "required": false,
+      "text": "Order Start Date",
+      "item": [
+        {
+          "type": "date",
+          "required": false,
+          "linkId": "3.1",
+          "text": "Order start date, if different from date of order"
+        }
+      ]
+    },
+    {
+      "linkId": "4",
+      "type": "group",
+      "required": false,
+      "text": "Order Type",
+      "item": [
+        {
+          "type": "choice",
+          "code": [],
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "drop-down",
+                    "display": "Drop down"
+                  }
+                ],
+                "text": "Drop down"
+              }
+            }
+          ],
+          "required": true,
+          "linkId": "4.1",
+          "text": "Type of Order",
+          "answerOption": [
+            {
+              "valueCoding": {
+                "code": "initial",
+                "display": "Initial or original order"
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "renewal",
+                "display": "Renewal order"
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "revision",
+                "display": "Revision or change order"
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "other",
+                "display": "Other"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "5",
+      "type": "group",
+      "required": false,
+      "text": "Order",
+      "item": [
+        {
+          "type": "choice",
+          "code": [],
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "drop-down",
+                    "display": "Drop down"
+                  }
+                ],
+                "text": "Drop down"
+              }
+            }
+          ],
+          "required": false,
+          "linkId": "5.1",
+          "text": "Order: Indwelling catheters",
+          "answerOption": [
```

Comment:
Using valueset and it should also be prepopulated from the service request HCSPCS code.